charts/rancher-webhook/templates/deployment.yaml

```diff
@@ -50,3 +50,6 @@ spec:
         - name: tls
           mountPath: /tmp/k8s-webhook-server/serving-certs
       serviceAccountName: rancher-webhook
+      {{- if .Values.priorityClassName }}
+      priorityClassName: "{{.Values.priorityClassName}}"
```

Comment:
Per the [k8s docs](https://kubernetes.io/docs/concepts/scheduling-eviction/pod-priority-preemption/#pod-priority) I don't think that we want quotation marks here. I could be wrong though, don't know much about this field.